app/controllers/registrations_controller.rb

```diff
@@ -44,6 +44,7 @@ def update_first_user_permissions(resource)
 
     resource.add_role(:super_admin)
     resource.add_role(:trusted)
+    resource.skip_confirmation!
```

Comment:
This looks pretty reasonable. If they're `trusted` and a `super_admin` because they're the first user, it probably makes a lot of sense to just ignore the confirmation. When I set up my own Forem without SendGrid or anything like that after I first started here, I had to get the confirmation URL from the Rails logs from when it _tried_ to send the email. This looks like a much better solution.